src/integrationTest/java/uk/gov/hmcts/darts/arm/service/CleanupArmResponseFilesServiceIntTest.java

```diff
@@ -0,0 +1,677 @@
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.test.context.ActiveProfiles;
+import uk.gov.hmcts.darts.arm.api.ArmDataManagementApi;
+import uk.gov.hmcts.darts.arm.config.ArmDataManagementConfiguration;
+import uk.gov.hmcts.darts.arm.service.impl.CleanupArmResponseFilesServiceImpl;
+import uk.gov.hmcts.darts.authorisation.component.UserIdentity;
+import uk.gov.hmcts.darts.common.entity.ExternalObjectDirectoryEntity;
+import uk.gov.hmcts.darts.common.entity.HearingEntity;
+import uk.gov.hmcts.darts.common.entity.MediaEntity;
+import uk.gov.hmcts.darts.common.entity.UserAccountEntity;
+import uk.gov.hmcts.darts.common.helper.CurrentTimeHelper;
+import uk.gov.hmcts.darts.common.repository.ExternalLocationTypeRepository;
+import uk.gov.hmcts.darts.common.repository.ExternalObjectDirectoryRepository;
+import uk.gov.hmcts.darts.common.repository.ObjectRecordStatusRepository;
+import uk.gov.hmcts.darts.testutils.IntegrationBase;
+import uk.gov.hmcts.darts.testutils.data.MediaTestData;
+
+import java.time.LocalDate;
+import java.time.OffsetDateTime;
+import java.time.ZoneOffset;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+import java.util.UUID;
+
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.when;
+import static uk.gov.hmcts.darts.common.enums.ExternalLocationTypeEnum.ARM;
+import static uk.gov.hmcts.darts.common.enums.ObjectRecordStatusEnum.ARM_RESPONSE_CHECKSUM_VERIFICATION_FAILED;
+import static uk.gov.hmcts.darts.common.enums.ObjectRecordStatusEnum.ARM_RESPONSE_MANIFEST_FAILED;
+import static uk.gov.hmcts.darts.common.enums.ObjectRecordStatusEnum.ARM_RESPONSE_PROCESSING_FAILED;
+import static uk.gov.hmcts.darts.common.enums.ObjectRecordStatusEnum.STORED;
+
+@SpringBootTest
+@ActiveProfiles({"intTest", "h2db"})
+class CleanupArmResponseFilesServiceIntTest extends IntegrationBase {
+
+    private static final LocalDate HEARING_DATE = LocalDate.of(2023, 9, 26);
+
+    @Autowired
+    private ExternalObjectDirectoryRepository externalObjectDirectoryRepository;
+    @Autowired
+    private ObjectRecordStatusRepository objectRecordStatusRepository;
+    @Autowired
+    private ExternalLocationTypeRepository externalLocationTypeRepository;
+    @MockBean
+    private ArmDataManagementApi armDataManagementApi;
+    @MockBean
+    private UserIdentity userIdentity;
+    @MockBean
+    private ArmDataManagementConfiguration armDataManagementConfiguration;
+    @MockBean
+    private CurrentTimeHelper currentTimeHelper;
+
+    private CleanupArmResponseFilesService cleanupArmResponseFilesService;
+
+    @BeforeEach
+    void setupData() {
+
+        cleanupArmResponseFilesService = new CleanupArmResponseFilesServiceImpl(
```

Comment:
Pretty sure if you autowired the `CleanupArmResponseFilesService`, looking at the other Autowired and MockBean usage, I'd expect this to happen anyway without being explicit here